api/proto/info/info.proto

```diff
@@ -673,3 +676,10 @@ message ZInfoMsg {
   }
   google.protobuf.Timestamp atTimeStamp = 6;
 }
+
+// Information about running hypervisor and hardware capabilities
+message Capabilities {
+  string hypervisor = 1;
```

Comment:
This needs to be pretty well documented before we can merge it. For example, what does this string mean? ;-)